_posts/2019-11-18-release-retrospection.md

```diff
@@ -0,0 +1,79 @@
+Please see also  [my comment in the respective issue](https://github.com/TeamNewPipe/NewPipe/issues/2615#issuecomment-530445812).
+
+For more information, see also the [release page](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.17.2).
+
+
+## Changes in 0.17.3
+
+In this release, again many improvements have been added to NewPipe. Most notably, a new option has been added to clear all positions in playlists. Until then, you had to clear your entire watch history, however in most cases that's undesirable. Another useful addition is that NewPipe can now show hidden files in the file picker.
+
+Most of you probably know [invidio.us](https://invidio.us), a free/open-source alternative web frontend to YouTube. NewPipe can now understand URLs pointing to this service, which means you can open links to Invidious in NewPipe just like you can open links to YouTube. Also, links to `music.youtube.com` can now be opened in NewPipe.
+
+The most interesting bugs fixed in this release are an annoying exception caused by a small but important typo in the extractor. Furthermore, a bug in the playback of live streams has been resolved. Also, a bug producing performance problems has been fixed in the extractor.
+
+For more information, see also the [release page](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.17.3).
+
+
+## Changes in 0.17.4
+
+In 0.17.4, the accessibility for blind people has been improved a little. Thanks to [@fcnjd](https://github.com/fcnjd) for assisting [@theScrabi](https://github.com/theScrabi) with helpful advice while implementing these changes. Furthermore, the wording of the description of the "video download folder" has been improved grammar wise.
+
+Some bugs have been fixed regarding the download of videos from YouTube. Furthermore, some bugs related to scrolling have been resolved. The "clear search box" is now animated properly, and a lot of issues with the SoundCloud implementation could be fixed in this release, making this provider more reliable than ever before.
+
+A few minor changes involve an upgrade of two dependencies (ExoPlayer and OkHttp) as well as migrating to the Android X APIs (a rather large and notable change that shouldn't go unmentioned). 
+
+For more information, see also the [release page](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.17.4).
+
+
+## Upcoming release
+
+Soon Team NewPipe will be able to release version 0.18.0. We will post a release announcement here once ready. A few changes which we can announce already involve using new localization and downloader implementations introduced in the extractor as well as adding support for the Kotlin language, allowing the development of new features in this language.
```

Comment:
Also planned for this release is the [overhaul of the "What's New" section](https://github.com/TeamaNewPipe/NewPipe/pull/2309) including a proper sorted feed and subscription groups. Review is almost finished :)